examples/local-transactions-with-participants/local_transaction_participants.bal

```diff
@@ -0,0 +1,37 @@
+import ballerina/io;
+// Import transactions moudle in order to use participant annotation.
+import ballerina/transactions;
```

Comment:
Why do we need to import transactions package here?

Reply:
Sorry for the noise. Just now noticed the annotation.

Reply:
It's for `@transactions:Participant` annotation